extra/resources/HealthSMART.in

```diff
@@ -278,13 +277,16 @@ HealthSMART_validate() {
 
     init_smart
```

Comment:
It looks like the checks in this should only be done on level 10 too, which leaves nothing for 0, though we could add validation that certain parameters are either empty or integers. Maybe even test that any items in drives start with a slash